src/org/jitsi/impl/neomedia/rtp/remotebitrateestimator/RemoteBitrateEstimatorAbsSendTime.java

```diff
@@ -21,74 +21,83 @@
     private final static int kInterArrivalShift
-            =  kAbsSendTimeFraction + kAbsSendTimeInterArrivalUpshift;
+        = kAbsSendTimeFraction + kAbsSendTimeInterArrivalUpshift;
     private final static int kInitialProbingIntervalMs = 2000;
     private final static int kMinClusterSize = 4;
     private final static int kMaxProbePackets = 15;
     private final static int kExpectedNumberOfProbes = 3;
 
-    private static final double kTimestampToMs = 1000.0 /
-            (1 << kInterArrivalShift) ;
+    //@Todo In webrtc, kTimestampToMs = 1000.0 / (1 << kInterArrivalShift);
+    //Clarify difference.
+    private static final double kTimestampToMs = 1.0 / 90.0;
+
 
     private final Object critSect = new Object();
-    private  TreeMap<Long,Long> ssrcs_ = new TreeMap<Long, Long>();
-    private  ArrayList<Probe> probes_ = new ArrayList<>();
-    private  long totalProbesReceived;
-    private  long firstPacketTimeMs;
-    private  long lastUpdateMs;
+
+    private TreeMap<Long, Long> ssrcs_ = new TreeMap<Long, Long>();
+    private ArrayList<Probe> probes_ = new ArrayList<>();
+    private long processIntervalMs = kProcessIntervalMs;
+    private long lastProcessTime = -1L;
+    private long totalProbesReceived;
+    private long firstPacketTimeMs;
+    private long lastUpdateMs;
     private RemoteBitrateObserver observer_;
-    private AimdRateControl remoteRate  = new AimdRateControl();
+    private AimdRateControl remoteRate = new AimdRateControl();
     private InterArrival interArrival;
     private OveruseEstimator estimator;
     private OveruseDetector detector;
-    private RateStatistics incomingBitrate ;
+    private RateStatistics incomingBitrate;
     private boolean incomingBitrateInitialized;
     private AbsSendTimeEngine absoluteSendTimeEngine;
 
     public RemoteBitrateEstimatorAbsSendTime(RemoteBitrateObserver observer,
-                                           AbsSendTimeEngine absSendTimeEngine)
+                                             AbsSendTimeEngine absSendTimeEngine)
     {
         super(RTPPacketPredicate.INSTANCE);
         this.absoluteSendTimeEngine = absSendTimeEngine;
         this.observer_ = observer;
         this.interArrival = new InterArrival(90 * kTimestampGroupLengthMs,
-                kTimestampToMs,true);
+            kTimestampToMs, true);
         this.estimator = new OveruseEstimator(new OverUseDetectorOptions());
         this.detector = new OveruseDetector(new OverUseDetectorOptions());
-        this.incomingBitrate = new RateStatistics(kBitrateWindowMs,8000);
+        this.incomingBitrate = new RateStatistics(kBitrateWindowMs, 8000F);
         this.incomingBitrateInitialized = false;
         this.totalProbesReceived = 0;
         this.firstPacketTimeMs = -1;
         this.lastUpdateMs = -1;
-        logger.info("; RemoteBitrateEstimatorAbsSendTime: Instantiating.");
+        logger.info("; RemoteBitrateEstimatorAbsSendTime: Calling.");
```

Comment:
Can we change that to debug level please (with an if(logger.isDebugEnabled()) check).